src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.cs

```diff
@@ -72,6 +108,13 @@ private void CheckStart()
             }
         }
 
+        private static IPAddress GetAddressSnapshot(IPAddress address)
+        {
+            return address.AddressFamily == AddressFamily.InterNetwork ?
+                new IPAddress(address.GetAddressBytes()) :
```

Comment:
As part of moving this you also changed it from `new IPAddress(address.GetAddressBytes())` to `new IPAddress(address.Address)`.  Please revert that, and put back the pragma warning disable as well as the comment explaining why `Address` was being used.

Reply:
I actually changed this due to this the following review comments: 

https://github.com/dotnet/corefx/pull/30000/files/7a40a628d74b21bb198191ba3bdc28607c357017#diff-de6c2f6788dcbd74fb88f8f2e7654067R111

I've no problem changing this back but can you confirm this is what you want having seen the comments please.

Reply:
I see.  I'd prefer to leave it the way it was (cc: @rmkerr).  The API is presumably obsoleted because it doesn't make sense for IPv6, but there's nothing wrong with using it for IPv4, which is how it's used here, and it avoids unnecessarily allocating an array.  We explicitly changed it from using GetAddressBytes to using Address not that long ago:
https://github.com/dotnet/corefx/commit/046ecd27b14aab61f8c7a370778091d2255ed6c3#diff-de6c2f6788dcbd74fb88f8f2e7654067

Reply:
Cool, I'll change it back tonight.

Reply:
Thanks.  Sorry for the back and forth.

Reply:
That's fine by me. Thanks for explaining @stephentoub.